### DISCUSSION TITLE
NocoDB on ECS [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5864] [Discord Threads]
> 🙋‍♂️ **VincentLayer**  
#### Anyone would know what might be the cause? No log too. The instance just terminate on booting up.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1103542786025853029/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### which version are you using?  
#  
> 🙋‍♂️ **merckxite**  
#### Hi <@495466997304262658>, I'm working on this with Vincent, we're using latest  
#  
> 🙋‍♂️ **merckxite**  
#### We spent some more time debugging and worked out it was an issue with our DB config. Unsure why it killed the logs entirely and resulted in that behaviour but we are happy to be up and running 😀  
#  
> 🙋‍♂️ **wkw**  
#### can u share more wt changes did you make?  
#  
> 🙋‍♂️ **merckxite**  
#### In the beginning we were relying on the autogenerated SQLite DB. This was resulting in restarts so we added a volume as we thought that this could have been the issue. We then tried hooking up to an RDS instance, which didn't work initially but we worked out that we had messed up some of the security group config. After we resolved that the instance booted up and performed as expected  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1103542786227187775)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
